.env

```diff
@@ -17,4 +17,5 @@ REACT_APP_DUAL_STAKING_LIST_DEFAULT_URL="https://unpkg.com/quickswap-default-sta
 REACT_APP_SYRUP_LIST_DEFAULT_URL="https://unpkg.com/quickswap-default-staking-list-address@latest/build/quickswap-default.syrups.json"
 REACT_APP_ADS_LIST_DEFAULT_URL="https://unpkg.com/quickswap-ads-list@latest/build/quickswap-ads-list.json"
 REACT_APP_LEGACY_APP_URL="https://legacy.quickswap.exchange"
-REACT_APP_MARKET_API_URL="https://sls.market.xyz/api"
\ No newline at end of file
+REACT_APP_MARKET_API_URL="https://sls.market.xyz/api"
+REACT_APP_MARKET_RPC="https://snowy-maximum-telescope.matic.discover.quiknode.pro/d0248e72f144a330566fe85fb4f920cfa2e1a3dc/"
```

Comment:
What address is this

Reply:
We need to use dedicated RPC in order to load market with at least 10 assets and I created a free quicknode account and used this RPC to init market SDK.